app/controllers/stripe-setup/check-org-details/post.controller.js

```diff
@@ -39,6 +42,14 @@ module.exports = (req, res, next) => {
 
     return response(req, res, 'stripe-setup/check-org-details/index', data)
   }
+
+  const credential = getCredentialByExternalId(req.account, req.params.credentialId)
+
+  if (confirmOrgDetails === 'yes') {
+    return res.redirect(303, formatAccountPathsFor(paths.account.yourPsp.index, req.account.external_id, credential.external_id))
```

Comment:
Can we also mark the stripe setup task for organisation details as `completed` if 'yes' is selected ? or is it planned for next PR ?

Reply:
Good spot - I have just updated it, please take a look.